src/vos/evtree.c

```diff
@@ -2948,18 +2939,12 @@ evt_sdist_split(struct evt_context *tcx, bool leaf, struct evt_node *nd_src,
 		nr += delta;
 		if (nr == boundary)
 			break;
-		entry_src = evt_node_entry_at(tcx, nd_src, nr);
-		evt_rect_read(&rtmp, &entry_src->ne_rect);
+		evt_node_rect_read_at(tcx, nd_src, nr, &rtmp);
 		dist = evt_mbr_dist(&mbr, &rtmp);
 	} while ((dist > 0) == cond);
 
 done:
-	entry_src = evt_node_entry_at(tcx, nd_src, nr);
-	entry_dst = evt_node_entry_at(tcx, nd_dst, 0);
-	memcpy(entry_dst, entry_src, sizeof(*entry_dst) * (nd_src->tn_nr - nr));
-
-	nd_dst->tn_nr = nd_src->tn_nr - nr;
-	nd_src->tn_nr = nr;
+	evt_split_common(tcx, leaf, nd_src, nd_dst, nr);
```

Comment:
(style)  'nd' may be misspelled - perhaps 'and'?
(style)  'nd' may be misspelled - perhaps 'and'?